auth-service/src/main/kotlin/uk/gov/communities/delta/auth/services/UserService.kt

```diff
@@ -300,6 +300,23 @@ class UserService(
         }
     }
 
+    suspend fun resetMfaToken(userToReset: LdapUser, session: OAuthSession, call: ApplicationCall) {
+        // the TOTP secret is stored in the "unixHomeDirectory" Active Directory field
+        try {
+            val modificationItem = ModificationItem(DirContext.REMOVE_ATTRIBUTE, BasicAttribute("unixHomeDirectory", null))
+            val modificationArray = arrayOf(modificationItem)
+            ldapServiceUserBind.useServiceUserBind {
+                it.modifyAttributes(userToReset.dn, modificationArray)
+                logger.atInfo().addKeyValue("unixHomeDirectory", "").log("MFA token deleted")
+            }
+        } catch (e: Exception) {
+            logger.atError().addKeyValue("unixHomeDirectory", "").log("Error deleted MFA token", e)
```

Comment:
Again minor, but I'd either get rid of the try catch, or at least log the username of the user we're updating. If you were coming to investigate this error you'd already know we're setting "unixHomeDirectory" to be empty from the stack trace (we also know the target user, but only because the controller logs it, it wouldn't be part of this message)